orchestrators/airflow/run_parallel_delegated_operations.py

```diff
@@ -0,0 +1,75 @@
+"""
+Airflow DAG that executes FiftyOne delegated operations in parallel.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+# pylint: disable=import-error
+import asyncio
+import logging
+import psutil
+import re
+import traceback
+
+from airflow.operators.python import task
+from airflow.decorators import dag, task
+from datetime import datetime
+from fiftyone.operators.delegated import DelegatedOperationService
+from fiftyone.operators.executor import ExecutionRunState, ExecutionResult
+
+logger = logging.getLogger(__name__)
+svc = DelegatedOperationService()
+
+
+@dag(start_date=datetime.now(),
```

Comment:
Have you taken a look a [SubDags](https://docs.astronomer.io/learn/subdags) it might make this cleaner in the UI...?